packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -3930,8 +3945,7 @@ class SemanticsConfiguration {
   ///
   /// An empty configuration doesn't contain any semantic information that it
   /// wants to contribute to the semantics tree.
-  bool get hasBeenAnnotated => _hasBeenAnnotated;
-  bool _hasBeenAnnotated = false;
+  bool hasBeenAnnotated = false;
```

Comment:
I have concerns about making this a public writable API... (see my other comment)